tests/rust-integration-tests/runtimetest/src/main.rs

```diff
@@ -38,12 +19,17 @@ fn get_spec() -> Spec {
 }
 
 fn main() {
-    let opts: Opts = Opts::parse();
     let spec = get_spec();
-
-    match opts.command {
-        SubCommand::ReadonlyPaths => tests::validate_readonly_paths(&spec),
-        SubCommand::SetHostName => tests::validate_hostname(&spec),
-        SubCommand::MountsRecursive => tests::validate_mounts_recursive(&spec),
+    let args: Vec<String> = env::args().collect();
+    let execute_test = match args.get(1) {
+        Some(execute_test) => execute_test.to_string(),
+        None => return eprintln!("error due to execute test flag not found"),
```

Comment:
nitpick : instead of `test flag` can we make it `test name` as it is not technically a flag . Same for the error message below this.